flutter/pubspec.yaml

```diff
@@ -13,16 +13,16 @@ dependencies:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  sentry: ">=4.0.0 <5.0.0"
-  package_info: ^0.4.0
+  # this should point to pub
+  sentry: ^4.0.5
```

Comment:
would not be better to keep between compatible versions?

Reply:
It has to be changed anyway before you release this even if I change it to `">=4.0.0 <5.0.0"`. As Sentry should probably be `5.0.0-prerelease.0` and at that time there is no sentry with a `5.0.0` release. There's also no reason to release a `sentry_flutter` because it's not null safe yet.

Reply:
Or am I overlooking something?

Reply:
we can discuss versioning in another PR, lets merge it, my question was more about, why changing from `>= ... <` to `^...`